<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraDemoClass</class>
 <widget class="QMainWindow" name="CameraDemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1338</width>
    <height>794</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CameraDemo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3" stretch="8,1">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="title">
          <string>相机原图</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="labelImage">
            <property name="font">
             <font>
              <family>AcadEref</family>
              <pointsize>28</pointsize>
             </font>
            </property>
            <property name="text">
             <string>原图</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="title">
          <string>处理后的图像</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="labelImageProcess">
            <property name="font">
             <font>
              <family>AcadEref</family>
              <pointsize>28</pointsize>
             </font>
            </property>
            <property name="text">
             <string>处理后的图</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,2,2">
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="title">
          <string>控制按钮</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="1" colspan="2">
           <widget class="QPushButton" name="pushButton_6">
            <property name="text">
             <string>xxx</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QPushButton" name="pushButtonCloseCamera">
            <property name="text">
             <string>关闭相机</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButtonTakePic">
            <property name="text">
             <string>拍照</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButtonOpenCamera">
            <property name="text">
             <string>打开相机</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QPushButton" name="pushButtonExit">
            <property name="text">
             <string>退出</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="pushButtonOpenClose">
            <property name="text">
             <string>打开白平衡</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="comboBoxMethod"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>处理方法</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="title">
          <string>串口</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>串口号</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBox"/>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButtonOpenPort">
            <property name="text">
             <string>打开串口</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButtonClosePort">
            <property name="text">
             <string>关闭串口</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="title">
          <string>信息显示</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEditDistance"/>
          </item>
          <item row="1" column="1">
           <widget class="QTextEdit" name="textEditMsgShow"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>距离</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>信息显示</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1338</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CameraDemo.qrc"/>
 </resources>
 <connections/>
</ui>
